### PR TITLE
use sequential wrapper to implement the sequential model

### PR DESCRIPTION
**Pull request by yezhem**

### CODE REVIEW

File: mlora/model_llama.py [hunk 1]
```diff
@@ -36,6 +36,23 @@ def forward(self, data: torch.Tensor) -> torch.Tensor:
...
+        self.norm_eps_ = eps
+        self.weight_ = weight
+
+    def _norm(self, data: torch.Tensor) -> torch.Tensor:
+        return data * torch.rsqrt(+ self.norm_eps_)
+
+    def forward(self, data: torch.Tensor) -> torch.Tensor:
+        input_dtype = data.dtype
+        v = data.to(torch.float32).pow(2).mean(-1, keepdim=True)
+        data = data * torch.rsqrt(v + self.norm_eps_)
```
Comment (merlintang) [inline mlora/model_llama.py:51]: can we leave comments here to explain why we need to do this?

**Review by merlintang — COMMENTED** [review]
does this apply to other model like ChatGLM?

**Comment by yezhem** [comment]
> does this apply to other model like ChatGLM?

No, The chatglm interface also requires special adaptation, the approach is the same.  The baseline pipeline framework's inability to handle inherited `nn.Sequential` classes, ref: https://github.com/pytorch/pytorch/issues/19808, there is no universal method for model conversion.

**Comment by yezhem** [comment]
@waitfor-night can help develop the feature for chatglm model.

File: mlora/model_llama.py (hunk 1 above)
Comment (merlintang) [reply mlora/model_llama.py:51]: pls update this point

**Review by merlintang — APPROVED** [review]
pls fix this and LGTM

**Pull request merged**